helper/create_new_user.py

```diff
@@ -97,5 +92,15 @@ def create_test_data(user_id: int):
 
 
 if __name__ == "__main__":
-    user_id = int(sys.argv[1])
-    create_user_data(user_id)
+    try:
+        test_user_id = int(sys.argv[1])

```

Comment:
If the first argument is e.g. -1 then that should pass this try-catch block, right? Perhaps we also want a range check or something too? Or does nothing particularly bad happen?

Reply:
Right. But I don't think that there is a range we can use. In the end, this is a script just for testing, in case we need to create a user manually with some data already. Never ever to be used in production

Reply:
Ah ok, indeed no problem then